.eslintignore

```diff
@@ -0,0 +1,8 @@
+build/
+coverage/
+assets/
+node_modules/
+.grunt/
+
+Gruntfile.js
```

Comment:
Absolutely. However I added `Gruntfile.js` this to `.eslintignore` and not to `.gitignore` :wink:. I'll keep this in here, ok?